### DISCUSSION TITLE
"Previous Literature" and Methodology

### DISCUSSION THREAD

**Discussion by michaelweinold** [Depart-de-Sentier/sentier.dev#16] [General]
...I will collect some related literature here:

**Comment by michaelweinold** [comment]
@cmutel, are you already familiar with the work of [Nakamura and Kondo on (non-linear) model-based LCA](https://doi.org/10.1162/108819802320971632)?

<img width="500" alt="Screenshot 2024-04-13 at 12 12 37" src="https://github.com/Depart-de-Sentier/sentier.dev/assets/23102087/05e4019b-ae3b-4a73-b951-48c1b095ba82">

**Comment by michaelweinold** [comment]
Section **9.1 Non-linear models** in [Computational Structure of Life-Cycle Assessment](http://link.springer.com/10.1007/978-94-015-9900-9) discusses non-linear production models and solutions to the inventory problem.

**Reply by cmutel** [reply]
For those of us who don't have a spare CHF 140 in our pockets, could you summarize or upload the key page(s)?

**Reply by michaelweinold** [reply]
I would suggest those interested in this chapter violate copyright on their own, if they don't have legal access 🙈
[The major shadow libraries](https://en.wikipedia.org/wiki/Shadow_library) all have the pdf of this book on file.

**Reply by cmutel** [reply]
Indeed they do. Not really worth it though - they claim all non-linear relationships would have to be solved simultaneously, which is indeed hard, and conclude that:

> The focus of LCA is system-wide on a microscopic level. This can only be achieved on the expense of ignoring certain details, for instance, non-linearities. Including non-linear relationships is likely to imply a more modest ambition with respect to either the system-wide character or the microscopic character.
